sdk/monitor/azquery/autorest.md

```diff
@@ -195,24 +224,23 @@ directive:
   - from: swagger-document
     where: $.parameters.OrderByParameter
     transform: $["x-ms-client-name"] = "OrderBy"
+  - from: swagger-document
+    where: $.parameters.RollUpByParameter
+    transform: $["x-ms-client-name"] = "RollUpBy"
+
 
   # change type of MetricsClientQueryResourceOptions.Aggregation from *string to []*AggregationType
   - from: options.go
     where: $
-    transform: return $.replace(/Aggregation \*string/g, "Aggregation []*AggregationType");
-  - from: 
-        - metrics_client.go
-        - metricsbatch_client.go
+    transform: return $.replace(/MetricsClientQueryResourceOptions struct {\n\t(?:\/\/.*\s)+\tAggregation \*string/g, "MetricsClientQueryResourceOptions struct {\n	// The list of aggregation types to retrieve\n  Aggregation []*AggregationType");
```

Comment:
Why do this for `MetricsClientQueryResourceOptions` but not `MetricsBatchClientQueryBatchOptions`?

Reply:
Changing the type from its original string to []*AggregationType is added convenience to the user, but there's potential for issues in the future. I've gotten more conservative in my design decisions, so I would prefer to stick to the swagger.

MetricsClientQueryResourceOptions is already GA so can't change it, but I can modify it for MetricsBatchClientQueryBatchOptions.

Reply:
That makes sense but gives us two different types for the same kind of value. Is that inconsistency okay? (I don't have a strong opinion)